release/release_gold.sh

```diff
@@ -2,8 +2,33 @@
 
 source _liferay_common.sh
 
+RELEASE_PROPERTIES_FILE="release-data/release.properties"
```

Comment:
We will need to add the logic to get the release.properties file from releases.liferay.com/dxp/<version>.